types/redux-form/lib/reduxForm.d.ts

```diff
@@ -18,15 +18,18 @@ import {
 } from "../index";
 
 export type FormSubmitHandler<FormData = {}, P = {}> =
-    (values: Partial<FormData>, dispatch: Dispatch<any>, props: P) => void | FormErrors<FormData> | Promise<any>;
+    (values: Partial<FormData>, dispatch?: Dispatch<any>, props?: P) => void | FormErrors<FormData> | Promise<any>;
```

Comment:
Why dispatch and props are optional?

Reply:
It's mistake 😳, thank you. I'v made them not optional in 7b6e78e